### PR TITLE
Problem detail

### PR DESCRIPTION
**Pull request by geniusgordon**

### CODE REVIEW

File: problem/views.py [hunk 1]
```diff
@@ -45,21 +55,20 @@ def volume(request):
...
-      'input': 'a, b <= 100000000',
-      'output': 'a+b',
-      'samp_input': '1 2\n4 5',
-      'samp_output': '3\n9\n',
-      'tag': [''],
-      'testcase': [{'num': 1, 'time': 1, 'memory': 32}, {'num': 2, 'time': 3, 'memory': 100}],
-    }
-    return render(request, 'problem/detail.html', p)
+def detail(request, pid):
+    print pid
```
Comment (nickelcheng) [inline problem/views.py]: Why do you print here?

File: problem/templates/problem/problem.html [hunk 2]
```diff
@@ -96,7 +100,7 @@ <h4 class='modal-title'>Limits</h4>
   <h3>Tags</h3>
   <div id='tags' style='float:left;width:60%'>
     {% for t in tag %}
```
Comment (nickelcheng) [inline problem/templates/problem/problem.html]: I prefer {% for tag in tags %}

**Comment by nickelcheng** [comment]
This PR is ok for me. @CHIAHSINHOU please test this.

**Comment by CHIAHSINHOU** [comment]
There seems something wrong.

**Comment by geniusgordon** [comment]
是不是沒有migrate
我在我的電腦沒有這個error

**Comment by CHIAHSINHOU** [comment]
我有砍掉DB 然後重新install過 也下了 python manage.py migrate
還是出現這個錯誤訊息

**Comment by geniusgordon** [comment]
這是哪個頁面出現的error

**Comment by CHIAHSINHOU** [comment]
不管打哪個頁面都會出現

**Comment by geniusgordon** [comment]
有檔案沒add到這個branch
現在應該OK了~

**Comment by nickelcheng** [comment]
@CHIAHSINHOU please test this again.

**Comment by CHIAHSINHOU** [comment]
It's ok for me.

**Comment by bruce3557** [comment]
+1, going to launch

**Pull request merged**